Feature/mg connect database

glutengo opened the pull request.

glutengo committed:
connect real database and add distinction between mock, group 4 and group 5

glutengo committed:
add docs and fix startup and encoding

glutengo committed:
connect more real data for group 4

miiyuutou reviewed (approved):
Ich habe mir die Datenbanken-Klassen nicht im Detail angeschaut, aber vielleicht könnte man eine Oberklasse für diese Klassen einführen? Hängt dann natürlich davon ab wie viel redundanter Code da tatsächlich ist.

glutengo committed:
fix review issues

- add superclass for database classes
- fix some naming errors in group5database

glutengo commented:
> Ich habe mir die Datenbanken-Klassen nicht im Detail angeschaut, aber vielleicht könnte man eine Oberklasse für diese Klassen einführen? Hängt dann natürlich davon ab wie viel redundanter Code da tatsächlich ist.

@miiyuutou besonders viel war nicht redundant, aber ich habe zumindest den constructor und die close methode in eine Oberklasse ausgelagert. Ich merge dann, wenn https://github.com/Sentiments-of-Bundestag/graphenauswertung/pull/13 gemerged ist.

glutengo committed:
align mock data and group4 data, use eigenvector calculation of pagerank
